### DISCUSSION TITLE
HLA-II allele format

### DISCUSSION THREAD

**Discussion by martinez14recerca** [griffithlab/pVACtools#1124] [General]
Hello,
I have been using pVACbind for HLA class I neoantigen prediction. I have recently started analyzing class II and I have encountered several problems to indicate the HLA-II allele format, after checking the list of valid alleles and the format. As an example, if I have a patient with the following HLA (`HLA-DRB1*15:01, HLA-DRB1*07:01, HLA-DQA1*01:02-HLA-DQB1*06:02, HLA-DQA1*02:01-HLA-DQB1*02:02, HLA-DPA1*02:01-HLA-DPB1*02:01, HLA-DPA1*01:03-HLA-DPB1*04:02`), the programm does not recognise it. I would like to know how to address this problem.

Kind regards.

**Comment by susannasiebert** [comment]
Thank you for your interest in pVACtools. I'm happy to help you with this issue.

Class II alleles are generally encoded without the `HLA-` prefix. Please enter your alleles exactly as they are denoted in the output from the `pvacseq valid_alleles` command. For you list this would be: `DRB1*15:01,DRB1*07:01,DQA1*01:02-DQB1*06:02,DQA1*02:01-DQB1*02:02,DPA1*02:01-DPB1*02:01,DPA1*01:03-DPB1*04:02`.

Please let me know if this fixes your problem.

**Reply by martinez14recerca** [reply]
Hello,
Trying this option I get the following output: 
```
pvacbind run /home/immunocart/ANALISISTCR/LLC_001_II.fasta Test20240711 HLA-DRB1*15:01,HLA-DRB1*07:01,HLA-DQA1*01:02-HLA-DQB1*06:02,HLA-DQA1*02:01-HLA-DQB1*02:02,HLA-DPA1*02:01-HLA-DPB1*02:01,HLA-DPA1*01:03-HLA-DPB1*04:02 all /home/immunocart/ANALISISTCR/20240711TESTPVACBIND_001_II -e2 11,12,13,14,15,16,17,18,19,20
Allele HLA-DPA1*01:03-HLA-DPB1*04:02 not valid. Skipping.
Allele HLA-DPA1*02:01-HLA-DPB1*02:01 not valid. Skipping.
Allele HLA-DQA1*01:02-HLA-DQB1*06:02 not valid. Skipping.
Allele HLA-DQA1*02:01-HLA-DQB1*02:02 not valid. Skipping.
Allele HLA-DRB1*07:01 not valid. Skipping.
Allele HLA-DRB1*15:01 not valid. Skipping.
No MHC class I alleles chosen. Skipping MHC class I predictions.
No MHC class II alleles chosen. Skipping MHC class II predictions.
```
It seems it does not recognise the allele format. Do you have any other suggestion to try? Kind regards.

**Reply by susannasiebert** [reply]
Your new command still includes the HLA- prefix in each allele. Please copy and paste the alleles exactly like I posted them above.

**Reply by martinez14recerca** [reply]
Thank you very much for your suggestion. Now it is running successfully.
Kind regards.
